plugin/src/main/kotlin/com/github/triplet/gradle/play/tasks/PublishApk.kt

```diff
@@ -24,8 +25,8 @@ open class PublishApk : PlayPublishPackageBase() {
     @get:PathSensitive(PathSensitivity.RELATIVE)
     @get:InputFiles
     internal val inputApks by lazy {
-        // TODO: If we take a customizable folder, we can fix #233, #227
-        variant.outputs.filterIsInstance<ApkVariantOutput>().map { it.outputFile }
+        extension.buildInputFolder?.listFiles(ApkFileFilter)?.toList()
+                ?: variant.outputs.filterIsInstance<ApkVariantOutput>().map { it.outputFile }
```

Comment:
We should probably be a bit more specific here. I would think if the user defines a folder through this property, we should fail the build if that folder is empty or does not contain any apk files. Why else would the user specify the folder. We should not fall back to the variant output in that case. In the current implementation we could have the following situation:

- The user defines a build folder
- that folder does not exist (`listFiles(Filter)` returns `null`)
- we would fall back to the variant output, which is probably not what the user expects

So we should rather have the following logic:
- If `buildInputFolder` is not null, and contains apks, upload them
- If `buildInputFolder` is not null, but does not exist or there are no apks, fail the build
- If `buildInputFolder` is null, upload apks from variant output